src/test/java/com/google/firebase/auth/UserRecordTest.java

```diff
@@ -9,13 +9,15 @@
 import com.google.api.client.json.JsonFactory;
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableMap;
+import com.google.firebase.auth.UserRecord.UpdateRequest;
```

Comment:
Changes in this file seem unnecessary (just imports)?

On the other hand, why don't we have unit tests for the change in `UserRecord` class?